examples/inputs/default_fin_config_ldes.yaml

```diff
@@ -13,18 +13,18 @@ system_costs:
     - 0
   om_capacity:
     - 0
-  om_batt_fixed_cost: 42.888
+  om_batt_fixed_cost: 42.888 # VRDB fixed cost
```

Comment:
I see this as a duplicate of `battery_om_per_kw` and should probably be set to zero here. Leaving this value was an error on my part. Again, we need to get away from multiple financial interfaces